Gemfile.lock

```diff
@@ -1,31 +1,31 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionmailer (3.2.22.4)
-      actionpack (= 3.2.22.4)
+    actionmailer (3.2.22.5)
```

Comment:
So, here, we don't really want to submit a new `Gemfile.lock` -- do you think you could remove that file from the commit using this guide? https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git